### PR TITLE
Add rendering for Mary multi-asset Values

### PR DESCRIPTION
**Pull request by intricate**

### CODE REVIEW

File: cardano-cli/test/Test/Cli/MultiAssetParsing.hs [hunk 1]
```diff
@@ -3,171 +3,41 @@
...
-prop_lexValue_fullySpecified :: Property
-prop_lexValue_fullySpecified =
+prop_roundtrip_Value_parse_render :: Property
+prop_roundtrip_Value_parse_render =
   property $ do
-    (input, expectedOutput) <- forAll genValueTokenFullySpecified
-    case lex valueTokenFullySpecified input of
-      Left pe -> failWith Nothing $ show pe
-      Right token -> token === expectedOutput
+    value <- forAll $ filter (not . null . valueToList) genValueDefault
```
Comment (intricate) [inline cardano-cli/test/Test/Cli/MultiAssetParsing.hs:24]: Ah, actually maybe I should support passing a `Range` to the generator. Either that or we could support parsing an empty string as an empty `Value`, but I'm not sure that's appropriate.

File: cardano-api/test/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2019 Input Output (Hong Kong) Ltd.
```
Comment (Jimbo4350) [inline cardano-api/test/LICENSE:190]: Should be 2020

File: cardano-api/test/NOTICE [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Copyright 2019 Input Output (Hong Kong) Ltd.
```
Comment (Jimbo4350) [inline cardano-api/test/NOTICE:1]: Should be 2020

File: cardano-cli/src/Cardano/CLI/Mary/RenderValue.hs [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+--
+-- Note that this textual representation can be parsed by 'parseValue'.
+renderValue :: RenderValueOptions -> Value -> Text
+renderValue (RenderValueOptions renderAdaAssetId) v =
+    Text.intercalate
+      " + "
+      (map (renderAssetIdQuantityPair renderAdaAssetId) vals)
+  where
+    vals :: [(AssetId, Quantity)]
+    vals = valueToList v
```
Comment (dcoutts) [inline cardano-cli/src/Cardano/CLI/Mary/RenderValue.hs:50]:
```
renderValue (RenderValueOptions renderAdaAssetId) =
    Text.intercalate " + "
  . map (renderAssetIdQuantityPair renderAdaAssetId)
  . valueToList
```

**Review by dcoutts — APPROVED** [review]
LGTM so far. Is the wiring in the query going to be in this PR or a separate one?

**Comment by intricate** [comment]
> LGTM so far. Is the wiring in the query going to be in this PR or a separate one?

I'm going to do it a part of this PR.

**Pull request merged**